File: SortingSearching/m.cpp
#include <bits/stdc++.h>
using namespace std;

#define ll long long 

int main(){
    int people, flats;
    ll k;
    cin >> people >> flats >> k;
    vector<ll> want(people), available(flats);
    for(int i=0; i<people; i++)
        cin >> want[i];
     for(int i=0; i<flats; i++)
        cin >> available[i];
    sort(want.begin(),want.end());
    sort(available.begin(),available.end());

    people -= 1;
    ll alloted = 0;
    for(ll flat: want)
    {


    }

File: SortingSearching/m.py
n, m, k = map(int, input().strip().split())
desired = list(map(int, input().strip().split()))
actual = list(map(int, input().strip().split()))
alloted = 0
n = n-1
desired.sort()
actual.sort()

for flat in desired:
    if alloted == m:
        break
    for i in range(0, len(actual)):
        if(actual[i]-k <= flat <= actual[i]+k):
            alloted = alloted+1
            actual.pop(i)
            break
print(alloted)